libr/bin/format/elf/elf.c

```diff
@@ -4095,10 +4102,10 @@ static RVector* load_symbols_from_phdr(ELFOBJ *eo, int type) {
 	}
 
 	// we reserve room for 4096 and grow as needed.
-	const size_t initial_capacity = 4096;
-	RVector *ret = r_vector_new (sizeof (RBinElfSymbol), NULL, NULL);
-	if (!ret || !r_vector_reserve (ret, initial_capacity)) {
-		r_vector_free (ret);
+	// const size_t initial_capacity = 4096;
+	// RVector *ret = r_vector_new (sizeof (RBinElfSymbol), NULL, NULL);
+	RVecRBinElfSymbol *ret = RVecRBinElfSymbol_new ();
+	if (!ret) { //  || !r_vector_reserve (ret, initial_capacity))
```

Comment:
i wanted to make it work without extra optimizations but the tests are not passing yet, so ill keep that to avoid more moving pieces until the tests pass.

Reply:
i have the feeling that this number may be different with rvec compared to rvector, also the growing strategy is different

Reply:
probably right, but 4096 is a nice multiple of 2 so it's a safe bet anyway?